File: app/src/main/java/com/ikue/japanesedictionary/activities/SettingsActivity.java
package com.ikue.japanesedictionary.activities;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.ikue.japanesedictionary.R;
import com.ikue.japanesedictionary.fragments.SettingsFragment;

public class SettingsActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_settings);

        // Setup the toolbar
        Toolbar toolbar = findViewById(R.id.toolbar);
        toolbar.setTitle(R.string.settings_activity_title);
        setSupportActionBar(toolbar);
        getSupportActionBar().setHomeButtonEnabled(true);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);

        if (getFragmentManager().findFragmentById(R.id.container)==null) {
            getFragmentManager().beginTransaction()
                    .add(R.id.container,
                            new SettingsFragment()).commit();
        }
    }
}

File: app/src/main/java/com/ikue/japanesedictionary/activities/SettingsActivity.kt
package com.ikue.japanesedictionary.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.commit
import com.ikue.japanesedictionary.R
import com.ikue.japanesedictionary.databinding.SettingsBinding
import com.ikue.japanesedictionary.fragments.SettingsFragment

class SettingsActivity : AppCompatActivity() {
    private lateinit var binding: SettingsBinding

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = SettingsBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.toolbar.setTitle(R.string.settings_activity_title)
        setSupportActionBar(binding.toolbar)

        supportActionBar?.setHomeButtonEnabled(true)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        supportFragmentManager.commit { replace(R.id.settings_container, SettingsFragment()) }
    }
}